docs/pages/server-access/getting-started.mdx

```diff
@@ -133,7 +133,7 @@ configuration:
 Run the following command to create a user that can access the Teleport Web UI:
 
 ```code
-$ sudo tctl users add tele-admin --roles=editor,access --logins=root,ubuntu,ec2-user
+$ tctl users add tele-admin --roles=editor,access --logins=root,ubuntu,ec2-user
```

Comment:
```suggestion
$ tctl users add alice --roles=editor,access --logins=root,ubuntu,ec2-user
```

Might as well use a more generic username. I think this was a leftover from when we had a built-in admin role.

Reply:
This would require changing the user name across 17 instances, not just here.